extensions/workspace/watch/watcher.ts

```diff
@@ -143,10 +157,22 @@ export class Watcher {
     if (buildResults && buildResults.length) {
       return buildResults;
     }
-    logger.console(`${idStr} doesn't have a compiler, nothing to build`);
+    logger.debug(`${idStr} doesn't have a compiler, nothing to build`);
```

Comment:
this kind of info is actually important to see in the console.

Reply:
reverted